src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -230,35 +229,24 @@ const formConfig = {
           title: 'Special Circumstances',
           path: 'special-circumstances',
           uiSchema: {
-            'ui:description': specialCircumstancesDescription,
-            'view:suicidal': {
-              'ui:title': 'In crisis or thinking of suicide?'
-            },
-            'view:homeless': {
-              'ui:title': 'Homeless or at risk of becoming homeless?'
-            },
-            'view:extremeFinancialHardship': {
-              'ui:title': 'Suffering from extreme financial hardship?'
+            'ui:title': 'Homelessness',
+            'view:homelessness': {
+              'ui:title': 'Are you homeless or in danger of becoming homeless?',
+              'ui:widget': 'yesNo',
+              'ui:options': {
```

Comment:
Interesting, are these not default on the `yesNo` widget?

Reply:
Good question, I didn't look at the widget internals to find out. Probably worth double-checking, though.

Reply:
Yep, those are definitely the defaults 😛 
Removed the whole deal.

Reply:
Thanks for checking 👍